extension/popup/popup.html

```diff
@@ -1,9 +1,19 @@
 <html>
-
-<body>
-  <h1>somanytabs</h1>
-  <button id="all-btn">Collect all tabs</button>
-  <button id="group-btn">Collect group tabs</button>
+<head>
+  <link href="/tailwind.css" rel="stylesheet">
+</head>
+<body class="font-sans px-3 bg-gray-50 dark:bg-gray-800">
+  <h1 class="text-lg mt-3 dark:text-white">somanytabs</h1>
+  <div class="mt-4">
+    <label for="inputTitle" class="text-sm text-gray-700 dark:text-gray-200 mb-0.5">Bookmark name</label>
+    <input type="text" id="inputTitle" name="inputTitle" class="w-full p-1 my-2 mb-1 mt-0 relative rounded shadow">
```

Comment:
Usually html ids and class names use a dash like input-title but camel case is fine here as well I guess ;)